### ISSUE TITLE
Ability to choose ports/positions when splitting matrix across strings

### ISSUE THREAD

**Issue by digitydogs** [smeighan/xLights#3849]
**Is your feature request related to a problem? Please describe.**
I have a matrix which requires 2 ports to run. The second port is barely using any lights and I want to have other props on the port, but I need to have them before the 2nd half of the matrix due to layout and wiring.   

Currently you can only split a matrix with the sequential parts of the matrix at the start of the sequential ports with any additional props on those ports having to come after the matrix.

**Describe the solution you'd like**
A way to use the visualizer and a way to assign start controller/port/channel for matrix segments when a matrix spans more than one port. 

**Describe alternatives you've considered**
Running a lot of excess wire and using two max 485s in order to utilize the extra 350+ pixels worth of addressing on that port, or leaving the port underutilized. 

Port two still has about 380 more pixels it can drive at 40fps.
![image](https://github.com/smeighan/xLights/assets/41389583/72b88e8a-04d8-4521-a3b6-c3e256dec9f1)

This is the path the data line will currently have to take to add the proposed prop to the chain
![image](https://github.com/smeighan/xLights/assets/41389583/423c6a0e-2935-4003-8186-31ec79075e60)

This is the path I want the data line to be able to take
![image](https://github.com/smeighan/xLights/assets/41389583/e0f784a4-5a44-47b2-82e7-dbc895e4d569)

So basically what I want is the ability to do this. 

![image](https://github.com/smeighan/xLights/assets/41389583/42326c26-e902-49cc-8169-10ff0a1b263b)

**Comment by merryoncherry** [comment]
Hey @digitydogs, I agree that there's room for an enhancement here.  One thing you might consider as a workaround though is that the visualizer does allow you to rearrange the model location in the first string, so if you make the first string of the matrix the small one you can quite easily do this:

![image](https://github.com/smeighan/xLights/assets/102841136/67e0960c-0f7e-414e-b858-ce36ce78f3ad)

**Comment by digitydogs** [comment]
Now that is a very interesting quirk I was not aware of and will have to keep in mind.   Sadly with my luck If I go out and rewire my matrix they will add the ability for the second half to be repositioned that day!

**Comment by keithsw1111** [comment]
The visualiser was never meant to support every possible way of connecting things together ... and it has been expanded to handle more configurations over time but I dont see this one ever happening. Fortunately there are ways to configure this manually if you really must wire up your matrix this way ... its just that you will likely manually need to configure your controllers as well.
From you diagram there is not enough detail to know exactly how the matrix is wired so it is hard to tell you exactly how to do that. Sometimes it is possible with shadow models or 2 matrices in a group. You may be able to do it with a custom model with a large gap in the nodes (for the other models) and then 2 shadow models for the visualiser. The latter would almost certainly work. 
Without an actual copy of the models it is hard to show you how you would need to change things to make it work.

**Comment by keithsw1111** [comment]
Ok ... as I said if you post the layout I can likely show you how to achieve it using existing functionality but if you feel that way I guess thats your choice.
